docs/plugins/convert.rst

```diff
@@ -66,6 +66,10 @@ file. The available options are:
   default configuration) non-MP3 files over the maximum bitrate before adding
   them to your library.
   Default: ``no``.
+- **auto_keep**: As opposite to **auto**, import non transcoded versions of
+  your files but still convert them to **dest**. It uses the default format
+  you have defined in your config file.
+  Default: ``no``.
```

Comment:
Maybe this should be slightly more explicit that this does this _automatically on import_? In that sense, it is similar to (rather than solely different from) `auto`.